Mathlib/Probability/Distributions/Poisson.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2024 Josha Dekker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josha Dekker
+-/
+
+import Mathlib.Analysis.NormedSpace.Exponential
+import Mathlib.Analysis.SpecialFunctions.Exponential
+import Mathlib.Probability.Notation
+
+/-! # Poisson distributions over ℕ
+
+Define the Poisson measure over the natural numbers
+
+## Main definitions
+* `poissonPmfReal`: the function `λ x ↦ exp (- λ) * λ ^ x / x!`
+  for `x ∈ ℕ`, which is the probability density function of a Poisson distribution with
+  rate `λ > 0`.
+* `poissonPmf`: `ℝ≥0∞`-valued pdf,
+  `poissonPmf λ = ENNReal.ofReal (poissonPmfReal λ)`.
+* `poissonMeasure`: a Poisson measure on `ℕ`, parametrized by its rate `λ`.
+-/
+
+open scoped ENNReal NNReal
+
+open MeasureTheory Real Set Filter Topology
+
+namespace ProbabilityTheory
+
+section PoissonPmf
```

Comment:
Can you close the section somewhere sensible, like before the definition of the measure, and close the namespace at the end of the file?